CONTRIBUTIONS GUIDE.md

```diff
@@ -79,4 +79,59 @@ git push -u origin <your_branch_name>
 **15** Congrats !! You made your PR with desired changes. Now once the PR is reviewed, your PR will be merged to the original code base for everyone to see and use.
 
 <br>
+
+## Note
+
+To run your forked project, you need to pass in your **firebase config** data in `firebase.tsx` file under the firebse folder.
+
+![Screenshot from 2023-05-31 02-42-20](https://github.com/PiyushKalyanpy/GyanaGuru/assets/119070798/417dbf83-b823-4f01-bf1f-45eaa622f1e1)
+
+<br>
+Else it results in the following error:
+
+![Screenshot from 2023-05-29 18-45-06](https://github.com/PiyushKalyanpy/GyanaGuru/assets/119070798/d94e0911-4844-4854-9170-1fa37856b595)
+
+<br>
+
+To fix this follow the below steps:
+
+<br>
+
+**1.** Head on to the following link.
+ 
+```
+https://console.firebase.google.com/
+```
```

Comment:
Don't put this link in the code block, just put it as a link so that it is visible like a link. For example:

1. Head on to https://console.firebase.google.com